src/main/webapp/js/utils/form/relation-field.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { Component } from 'react';
+import _ from 'lodash';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { getEntityArrayByName } from '../../selectors';
+import SelectField from './select-field';
+
+class RelationField extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      options: [{ label: null, value: null }],
+    };
+  }
+
+  componentDidMount() {
+    const { entity, fieldConfig } = this.props;
+    const { relatedField } = fieldConfig;
+    const options = _.map(entity, row => ({ label: row[relatedField], value: row.id }));
```

Comment:
I think this should be done in render method without using a local state or even better create a new selcetor for this